locales/help_enu/gkhAncestorsCircle.html

```diff
@@ -4,31 +4,42 @@
 <meta charset="utf-8" />
 <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
 <link rel="stylesheet" href="styles.css" type="text/css" />
-<title>Ancestors circle</title>
+<title>Ancestors Circle</title>
 </head>
 <body>
-<h1>Ancestors circle</h1>
-<p><em>Ancestors circle</em> is a special-purpose diagram that shows all ancestors of selected person as sectors of concentric circles. Settings for
-this diagram are available at &quot;<a href="gkhOptions.html#circle">Diagrams\Ancestors circle</a>&quot; tab in &quot;Options&quot window. You can open &quot;Options&quot dialog using
-&quot;Edit\Options...&quot; menu. Settings on the specified tab allows you to set up the diagram appearance.</p>
-<p>To generate &quot;Ancestors circle&quot; diagram select a target entry in the persons list and then use &quot;Ancestors circle&quot; item in &quot;Pedigree&quot; menu.</p>
-<p>The following actions are available while you are viewing the diagram:
+<h1>Ancestors Circle</h1>
+<p>
+The <em>Ancestors circle</em> is a special-purpose diagram that shows all ancestors of the selected person as sectors of concentric circles. Settings for
+this diagram are available at &quot;<a href="gkhOptions.html#circle">Diagrams\Ancestors circle</a>&quot; tab in the &quot;Options&quot; window.
+You can open &quot;Options&quot; dialog using the
+&quot;Edit\Options...&quot; menu. Settings on the specified tab allows you to set up the diagram appearance.
+</p>
+<p>
+To generate an &quot;Ancestors circle&quot; diagram, select a person in the persons list and then use the &quot;Ancestors circle&quot; item in the
+&quot;Pedigree&quot; menu.
+</p>
+<p>
+The following actions are available while you are viewing the diagram:
 <ul>
-<li><kbd>[ Left ]</kbd> key updates diagram of father of selected person,</li>
-<li><kbd>[ Right ]</kbd> key updates diagram of mother of selected person,</li>
-<li><kbd>[ Up ]</kbd> key increases the number of displayed generations,</li>
-<li><kbd>[ Down ]</kbd> key decreases the number of displayed generations,</li>
-<li><kbd>[ Backspace ]</kbd> key returns to a previous person,</li>
-<li>click a sector to update diagram of selected person,</li>
-<li>doule click outside the diagram to change the current view mode to ancestors compositional analysis mode or revert the current mode back.</li>
+<li>The <kbd>[ Left ]</kbd> key updates the diagram of the father for the selected person.</li>
+<li>The <kbd>[ Right ]</kbd> key updates the diagram of the mother for the selected person.</li>
+<li>The <kbd>[ Up ]</kbd> key increases the number of displayed generations.</li>
+<li>The <kbd>[ Down ]</kbd> key decreases the number of displayed generations.</li>
+<li>The <kbd>[ Backspace ]</kbd> key returns to the previous person.</li>
+<li>Click a sector to update the diagram of selected person.</li>
+<li>Double-click outside the diagram to change the current view mode to ancestors compositional analysis mode or revert the current mode back.</li>
```

Comment:
Please note - here the changes are such that they are distinguishable - they can be seen separately, and here they can be studied and merged.